Gems/EMotionFX/Code/Source/Editor/PropertyWidgets/BlendSpaceMotionContainerHandler.cpp

```diff
@@ -77,7 +77,7 @@ namespace EMotionFX
             m_spinboxY->setDecimals(4);
             m_spinboxY->setRange(-FLT_MAX, FLT_MAX);
             m_spinboxY->setProperty("motionId", motionId.c_str());
-            m_spinboxX->setKeyboardTracking(false);
+            m_spinboxY->setKeyboardTracking(false);
             layoutY->addWidget(m_spinboxY);
 
             layout->addLayout(layoutY, row, column);
```

Comment:
I would personally rather see this duplicated code rolled up into a function (or lambda):
```cpp
const auto makeSpinbox = [row, &column, layout, motionId = motionId.c_str()](const QString& text, const QString& color)
{
    auto* axisLayout = new QHBoxLayout();
    axisLayout->setAlignment(Qt::AlignRight);
                                                                                                                        
    auto* axisLabel = new QLabel(text);
    axisLabel->setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
    axisLabel->setStyleSheet(QString("QLabel { font-weight: bold; color : %1; }").arg(color));
    axisLayout->addWidget(axisLabel);
                                                                                                                        
    auto* spinbox = new AzQtComponents::DoubleSpinBox();
    spinbox->setSingleStep(0.1);
    spinbox->setDecimals(4);
    spinbox->setRange(-FLT_MAX, FLT_MAX);
    spinbox->setProperty("motionId", motionId);
    spinbox->setKeyboardTracking(false);
    axisLayout->addWidget(spinbox);
                                                                                                                        
    layout->addLayout(axisLayout, row, column);
    column++;
                                                                                                                        
    return spinbox;
};
                                                                                                                        
m_spinboxX = makeSpinbox("X", "red");
                                                                                                                        
if (showYFields)
{
    m_spinboxY = makeSpinbox("Y", "green");
}
else
{
    m_spinboxY = nullptr;
}
```

Reply:
Changed - good suggestion, and thanks for the already prepared code!